pkg/command/service/measure.go

```diff
@@ -136,12 +132,7 @@ kperf service measure --perfix svc --range 1,200 --namespace ns --job 20
 
 			svcReadyTime := make([]float64, 0)
 
-			client, err := kubernetes.NewForConfig(cfg)
-			if err != nil {
-				return fmt.Errorf("failed to create k8s client%s\n", err)
-			}
-
-			nwclient, err := networkingv1alpha1.NewForConfig(cfg)
+			nwclient, err := p.NewNetworkingClient()
 			if err != nil {
 				return fmt.Errorf("failed to create serving client%s\n", err)
```

Comment:
please help to update the wrong error message, thanks!

Reply:
failed to create serving client --> failed to create networking client

Reply:
Fixed.